apps/organisasjon-forvalter/README.md

```diff
@@ -1,21 +1,28 @@
 ---
-layout: default
-title: Forvalter
-parent: Organisasjon
-grand_parent: Applikasjoner
+layout: default title: Forvalter parent: Organisasjon grand_parent: Applikasjoner
 ---
 
 # Organisasjon Forvalter
 
+Forvalter som oppretter og deretter deployer organisasjoner basert på innsendte kriterier. Håndterer også status per
```

Comment:
Hva menes med "deployer organisasjoner"?

Reply:
Når den legges på den EREG jenkins køen og blir tilgjengelig i miljøet, så den var namet `/deploy` i controller så gikk for det 🤓